packages/Notification/Notification.md

```diff
@@ -1,8 +1,14 @@
 **Use the "Open isolated" button above to view this component in full-width mode.**
 
-The `Notification` spans the entire width of the screen, and aligns the message with the page content.
+A banner that highlights important messages.
 
-By default, notifications will be displayed in the `instructional` variant.
+Notifications are used to let users know:
+
+- System status such as an error or outage
+- Their actions has triggered a change such as confirmation of a payment
```

Comment:
Change "has" to "have"

Result: "Their actions have triggered a change such as confirmation of a payment"